examples/flask/hello.py

```diff
@@ -34,7 +34,7 @@ def mask_event(eventmodel):
     return eventmodel
 
 moesif_settings = {
-    'APPLICATION_ID': 'Your Moesif Application Id',
+    'APPLICATION_ID': 'eyJhcHAiOiI3MTA6NzEyIiwidmVyIjoiMi4wIiwib3JnIjoiMjYyOjU0OSIsImlhdCI6MTY1NjYzMzYwMH0.CiU4oW_PuDXhLM8_scQktI_wrqZcu3GAFxUMb0DyQCc',
```

Comment:
@XinyuLuan you have checked in app id. Please self review PR after committing :-} - it helps  a lot.

Reply:
Yes, I will be careful next time, sorry